tests/unit/wait-for-promise-test.ts

```diff
@@ -1,17 +1,18 @@
+// @ts-ignore
```

Comment:
Same comment as above, we should ensure that the `qunit` module is properly typed

Reply:
Same as my answer above.

Reply:
> Same comment as above, we should ensure that the qunit module is properly typed

I hope get this done thurs/fri this week at latest

Reply:
We managed to figure out a stop-gap solution as part of #9